lib/archethic/account.ex

```diff
@@ -27,6 +19,8 @@ defmodule Archethic.Account do
   """
   @spec get_balance(Crypto.versioned_hash()) :: balance()
   def get_balance(address) when is_binary(address) do
+    # TODO
+    # replace with archethic.get inputs and returns utxo from inputs
```

Comment:
get_inputs is different that get_unspent_outputs

Reply:
yes, we calculate whether the input is spent or not, then build utxo.

Reply:
right